packages/client/src/LocClient.ts

```diff
@@ -1729,13 +1729,15 @@ export class AuthenticatedLocClient extends LocClient {
         return undefined;
     }
 
-    async openTransactionLoc(parameters: OpenPolkadotLocParams & BlockchainSubmissionParams ) {
-        const { locId, signer, callback } = parameters;
+    async openTransactionLoc(parameters: OpenPolkadotLocParams & BlockchainSubmissionParams & AutoPublish, requirePreOpen = true) {
```

Comment:
I think that `requirePreOpen` is redundant with `autoPublish`. It seems that their values are always the same, unless there are cases where `requirePreOpen` is not passed as an argument?

Reply:
Yes exactly, there are locations where `requirePreOpen` is not set and therefore always true, while `autoPublish` can be still be true or false, for [instance here](https://github.com/logion-network/logion-api/blob/main/packages/client/src/Loc.ts#L1409-L1420).